app/javascript/packs/components/hero_banner.tsx

```diff
@@ -35,7 +39,14 @@ export const HeroBanner: React.FC<Props> = (props: Props) => (
   <div className="flex flex-column w-full border-b-1 bg-maize-crayola">
     <div className="w-1/2 md:w-full md:py-4 sm:w-full sm:py-4 flex items-center text-center justify-center">
       <h1 className="text-5xl font-heading">
-        {props.title}
+        Share 
```

Comment:
can we have it only on home page ? it's not relevant to other pages. You can have a conditional here. Some pseudocode is as below -

```
{(page === "home") ? {<Typed ...>} : {props.title}}
```

Reply:
I know little react and no typescript so I have no idea what you are talking about. .. trying my best 😥

Reply:
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Something like this 😄

Reply:
Updated the link above